### ISSUE TITLE
na_ontap_quotas: reinitialize doesn't work for new quota

### ISSUE THREAD

**Issue by mamoep** [ansible-collections/netapp#108]
##### SUMMARY

I use na_ontap_quotas to create a qtree quota. I expect to create and reinitialize the quotas with one task. This doesn't work. I always need a second run to initialize the quota.

##### ISSUE TYPE
- Bug Report

##### COMPONENT NAME

na_ontap_quotas

##### ANSIBLE VERSION

```paste below
ansible 2.9.18
  config file = /ansible/private/ansible.cfg
  configured module search path = [u'/home/user/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Aug 13 2020, 02:51:10) [GCC 4.8.5 20150623 (Red Hat 4.8.5-39)]

```

##### CONFIGURATION

```paste below

```

##### OS / ENVIRONMENT

RHEL 7.9
ONTAP 9.7P12
Collection 21.5.0

##### STEPS TO REPRODUCE

Execute the playbook.

```yaml
  - name: Create and reinitialize Quota
    na_ontap_quotas:
      state: present
      volume: "{{ volume.name }}"
      quota_target: "/vol/{{ volume.name }}/{{ qtree.name }}"
      type: tree
      disk_limit: "{{ qtree.size }}"
      set_quota_status: True
      activate_quota_on_change: reinitialize
      vserver: "{{ vserver.name }}"
      <<: *login
    connection: local
```

##### EXPECTED RESULTS

Quota rule is created (visible in "quota policy rule show"), and initialized (visible in "quota report"). 

##### ACTUAL RESULTS

Quota rule is created (visible in "quota policy rule show"), but not initialized (not visible in "quota report"). Only if I run the playbook a second time, the initialization is done.

```paste below
task path: /ansible/private/playbooks/netapp/test/create_share.yml:128
<127.0.0.1> ESTABLISH LOCAL CONNECTION FOR USER: user
<127.0.0.1> EXEC /bin/sh -c 'echo ~user && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/user/.ansible/tmp `"&& mkdir "` echo /home/user/.ansible/tmp/ansible-tmp-1620111173.1-7445-223113219997491 `" && echo ansible-tmp-1620111173.1-7445-223113219997491="` echo /home/user/.ansible/tmp/ansible-tmp-1620111173.1-7445-223113219997491 `" ) && sleep 0'
Using module file /home/user/.ansible/collections/ansible_collections/netapp/ontap/plugins/modules/na_ontap_quotas.py
<127.0.0.1> PUT /home/user/.ansible/tmp/ansible-local-6916PUXM5T/tmpuFyuAJ TO /home/user/.ansible/tmp/ansible-tmp-1620111173.1-7445-223113219997491/AnsiballZ_na_ontap_quotas.py
<127.0.0.1> EXEC /bin/sh -c 'chmod u+x /home/user/.ansible/tmp/ansible-tmp-1620111173.1-7445-223113219997491/ /home/user/.ansible/tmp/ansible-tmp-1620111173.1-7445-223113219997491/AnsiballZ_na_ontap_quotas.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c '/usr/bin/python2 /home/user/.ansible/tmp/ansible-tmp-1620111173.1-7445-223113219997491/AnsiballZ_na_ontap_quotas.py && sleep 0'
<127.0.0.1> EXEC /bin/sh -c 'rm -f -r /home/user/.ansible/tmp/ansible-tmp-1620111173.1-7445-223113219997491/ > /dev/null 2>&1 && sleep 0'
changed: [localhost] => {
    "changed": true,
    "invocation": {
        "module_args": {
            "activate_quota_on_change": "reinitialize",
            "cert_filepath": null,
            "disk_limit": "5g",
            "feature_flags": {},
            "file_limit": null,
            "hostname": "filer.domain.local",
            "http_port": null,
            "https": true,
            "key_filepath": null,
            "ontapi": null,
            "password": "VALUE_SPECIFIED_IN_NO_LOG_PARAMETER",
            "perform_user_mapping": null,
            "policy": null,
            "qtree": "",
            "quota_target": "/vol/filer_volume/Test-19",
            "set_quota_status": true,
            "soft_disk_limit": null,
            "soft_file_limit": null,
            "state": "present",
            "threshold": null,
            "type": "tree",
            "use_rest": "auto",
            "username": "user",
            "validate_certs": false,
            "volume": "filer_volume",
            "vserver": "filer"
        }
    }
}
```

**Comment by lonico** [comment]
I could reproduce it.  By having an existing qtree and deleting the quota first.

**Comment by mamoep** [comment]
Thanks, for the quick fix! # 0b5fdef
Included in the 21.6.0 release
